src/app/Library/CrudPanel/Traits/Relationships.php

```diff
@@ -192,6 +192,27 @@ private function changeBelongsToNamesFromRelationshipToForeignKey($input, $field
         return $input;
     }
 
+    /**
```

Comment:
Maybe it'd make sense to move all these new Morph functions to a new CrudPanel trait... say `MorphRelationships`? 🤷‍♂️ What do you think @pxpm ?